File: CouchShopperAPI/CouchShopper.Data/DTOs/Response/Users/UserResponse.cs
using System;

namespace CouchShopper.Data.DTOs.Response.Users
{
    public class UserResponse
    {
        public string Username { get; set; }

        public string FullName { get; set; }

        public string Email { get; set; }

        public string Country { get; set; }

        public string City { get; set; }

        public string Address { get; set; }

        public string Phone { get; set; }

        public string ZipCode { get; set; }

    }
}

File: CouchShopperWeb/CouchShopperWeb/src/app/_models/Account/response/user-response.ts
export interface ActivationForm {
  activationCode:string
}
export interface LoginResponse {
  username: string;
  token: string;
  role:string
}

export interface UserIsActiveResponse {
  isActive: boolean
}

export interface UserResponse {
  username: string
  fullName: string
  email: string
  country: string
  city: string
  address: string
  phone: string
  zipCode: string
}
